### PR TITLE
Distinguishing States kata (ex-Measurements): add first task

### PR DESCRIPTION
**Pull request by tcNickolas**
Includes refactoring to move measurements library operations from Common to KatasLibrary

### CODE REVIEW

File: katas/content/index.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "multi_qubit_gates",
   "single_qubit_measurements",
   "multi_qubit_measurements",
+  "distinguishing_states",
```
Comment (tcNickolas) [inline katas/content/index.json:11]: TODO: remove before merge

File: katas/content/distinguishing_states/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "id": "distinguishing_states__overview",
+    "title": "Overview"
+})
+
+This kata is designed to get you familiar with the concept of measurements and using them for analyzing quantum states and with Q# programming.
+
+**This kata covers the following topics:**
+
+- single-qubit measurements,
+- distinguishing orthogonal and nonorthogonal states.
```
Comment (cesarzc) [inline katas/content/distinguishing_states/index.md:13]:
```suggestion
- distinguishing orthogonal and non-orthogonal states.
```

File: katas/content/distinguishing_states/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "../KatasLibrary.qs"
+    ]
+})
+
+
+@[section]({
+    "id": "distinguishing_states__nonorthogonal_states",
+    "title": "Distinguishing Nonorthogonal States"
+})
+
```
Comment (cesarzc) [inline katas/content/distinguishing_states/index.md:40]: Should a TODO be added here as well since this section is empty?

**Review by cesarzc — APPROVED** [review]

File: katas/content/distinguishing_states/index.md (hunk 3 above)
Comment (tcNickolas) [reply katas/content/distinguishing_states/index.md:40]: It's implicit :-) It should be enough to have one TODO in the end of the kata to remind us that it's incomplete

**Pull request merged**